### ISSUE TITLE
windows10 calculator is not fully accessible

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5260]
**Reported by joshknnd1982 on 2015-08-02 00:07**
When in the windows10 calculator if I do a calculation NVDA nex snapshot does not read the result. I am using the latest next snapshot of NVDA. Perhaps there is a way to use the windows7 calculator in windows10?

**Comment by bhavyashah** [comment]
@josephsl I believe you did some work on Windows 10 Calculator as part of the Windows 10 App Essentials add-on. If this is indeed so, could you please elaborate on the degree of accessibility it offers for said app, possibility of integrating that support into core, etc.?

**Comment by ehollig** [comment]
Closing as duplicate of #5760 as it contains more recent comments.
